### ISSUE TITLE
 __pyx_check_sizeof_voidp = 1 / (int)(SIZEOF_VOID_P == sizeof(void*)) A wheel is created for you to put on pypi.

### ISSUE THREAD

**Issue by RonDingDing** [siddhantgoel/streaming-form-data#44]
From line 200 of _parser.c there is a enum definition:
```c
#ifdef SIZEOF_VOID_P
    enum { __pyx_check_sizeof_voidp = 1 / (int)(SIZEOF_VOID_P == sizeof(void*)) };
  #endif
```
which, when SIZEOF_VOID_P != sizeof(void*)), will raise an error. I compile your code on windows mingw-w64 using ```python setup.py build --compiler=mingw32``` and get this error, which means SIZEOF_VOID_P is defined and SIZEOF_VOID_P != sizeof(void*). Is that a deliberate setting?

**Comment by RonDingDing** [comment]
Besides can you provide a windows wheel version? It is really discouraging that we can not use this library on windows.

**Comment by RonDingDing** [comment]
Well, please close the issue. I've built a wheel using msvc compiler.
[streaming_form_data-1.7.1-cp38-cp38-win_amd64.zip](https://github.com/siddhantgoel/streaming-form-data/files/5450773/streaming_form_data-1.7.1-cp38-cp38-win_amd64.zip)

**Comment by siddhantgoel** [comment]
> Is that a deliberate setting?

This file is auto-generated using Cython. If you come across a bug in the generated C code, the Cython maintainers would be able to help you better.

Regarding the Windows wheel, yes it has been on my todo list for a while now. I don't have access to a Windows machine which makes it tricky for me to build a Windows wheel. I would most likely have to integrate this "build and push to PyPI" process into Github actions, which I don't think is too trivial. I'll see if I can find some time for this over the next few days.

**Comment by siddhantgoel** [comment]
Actually publishing to PyPI from Github Actions might not be tricky after all. I'll try to wrap it up over the weekend.

**Comment by RonDingDing** [comment]
Thank you~

**Comment by siddhantgoel** [comment]
Platform wheels for Linux, macOS and Windows should be available from v1.8.0 onwards.
